live-editing/configs/TreeGridConfigGenerator.ts

```diff
@@ -186,8 +187,9 @@ export class TreeGridConfigGenerator implements IConfigGenerator {
         configs.push(new Config({
             additionalFiles: ["/src/app/data/utils.ts", "/src/app/tree-grid/data/employees-flat.ts"],
```

Comment:
The patch to the `TreeGridWithTransactionsComponent` should be added, so the StackBlitz sample properly works:
```suggestion
             additionalFiles: ["/src/app/data/utils.ts", "/src/app/tree-grid/data/employees-flat.ts", "/src/app/tree-grid/tree-grid-batch-editing/tree-grid-with-transactions.component.ts"],
```